### PR TITLE
Styled download button and added copy logic

### PR DESCRIPTION
**Pull request by maaktweluit**
Not so happy to replace `%` of padding with `px`, but the button was moving off screen :(

Fixed:
- Matched styling with figma
- Added logic to select and copy the link on click of the whole box

Not fixed:
- No visual feedback yet when the link gets copied ( only the link gets selected )
  - we can either add a tooltip or change the text ( temporarily

### CODE REVIEW

File: src/app/output/metadata/output-file-metadata.component.ts [hunk 1]
```diff
@@ -52,4 +52,14 @@ export class OutputFileMetadataComponent implements OnInit {
     this.downloadEvent.emit(event);
   }
 
+  onCopyLinkClicked(event) {
+    var target = event.target || event.srcElement || event.currentTarget;
+    console.log("Copy Link Clicked: ", target)
```
Comment (bidzyyys) [inline src/app/output/metadata/output-file-metadata.component.ts]: Do we need console output?

File: src/app/output/metadata/output-file-metadata.component.ts [hunk 2]
```diff
@@ -52,4 +52,14 @@ export class OutputFileMetadataComponent implements OnInit {
     this.downloadEvent.emit(event);
   }
 
+  onCopyLinkClicked(event) {
+    var target = event.target || event.srcElement || event.currentTarget;
```
Comment (bidzyyys) [inline src/app/output/metadata/output-file-metadata.component.ts]: Why not `const`?

File: src/app/output/metadata/output-file-metadata.component.ts [hunk 3]
```diff
@@ -52,4 +52,14 @@ export class OutputFileMetadataComponent implements OnInit {
...
+    console.log("Copy Link Clicked: ", target)
+    var range = document.createRange();
```
Comment (bidzyyys) [inline src/app/output/metadata/output-file-metadata.component.ts]: Why not `const`?

File: src/app/output/metadata/output-file-metadata.component.ts (hunk 1 above)
Comment (maaktweluit) [reply src/app/output/metadata/output-file-metadata.component.ts]: i'm used to having console output for all the things, makes finding bugs ( especially in transposed code ) easier.

Removed it for now, seems more consistent with the rest of the application :)

File: src/app/output/metadata/output-file-metadata.component.ts (hunk 3 above)
Comment (maaktweluit) [reply src/app/output/metadata/output-file-metadata.component.ts]: forgot it after copy / pasta / testing, too used to es5 still i guess ;)

File: src/app/output/metadata/output-file-metadata.component.ts (hunk 2 above)
Comment (maaktweluit) [reply src/app/output/metadata/output-file-metadata.component.ts]: fixed

**Pull request merged**